README.md

```diff
@@ -149,6 +149,9 @@ The commands outlined below should be executed within the "src" directory if you
 python3 das_cli.py <command> <subcommand> [options]
 ```
 
+- `--help`: Package help (global flag).
+- `--version`: Package Version.
```

Comment:
Shouldn't this have the same "global flag"  tag?

Reply:
you are right